specs/mrtcore/PrimaryLanguageOverride/PrimaryLanguageOverride.md

```diff
@@ -0,0 +1,190 @@
+# ApplicationLanguages
+
+This new type (`Microsoft.Windows.ApplicationModel.Resources.ApplicationLanguages`) is a wrapper for
+the OS `Windows.Globalization.ApplicationLanguages` type with the updated behavior for the
+`PrimaryLanguageOverride` property to be supported for un-packaged apps.
+
+# Background
+
+Changing the application language using
+[PrimaryLanguageOverride](https://learn.microsoft.com/uwp/api/windows.globalization.applicationlanguages.primarylanguageoverride)
+property from `Windows.Globalization.ApplicationLanguages` namespace is not supported for
+WindowsAppSDK un-packaged apps, only for packaged. To support language change for un-packaged
+applications `ApplicationLanguages` type is introduced in
+`Microsoft.Windows.ApplicationModel.Resources` MRTCore namespace.
+`Microsoft.Windows.ApplicationModel.Resources.ApplicationLanguages.PrimaryLanguageOverride` property
+is supported both for packaged and un-packaged WindowsAppSDK apps.
+
+This new type (`Microsoft.Windows.ApplicationModel.Resources.ApplicationLanguages`) is a wrapper for
+the OS type with the updated behavior for the `PrimaryLanguageOverride` to support un-packaged apps.
+`Microsoft.Windows.ApplicationModel.Resources.ApplicationLanguages` should serve as a replacement
+for the existing OS type (`Windows.Globalization.ApplicationLanguages.PrimaryLanguageOverride`).
+
+`GetLanguagesForUser(User)` method from `Windows.Globalization.ApplicationLanguages` is being
+intentionally left off due to not being relevant for WinAppSDK.
+
+# API Pages
```

Comment:
This is all copy/paste from existing docs? Or are there differences in here beyond a couple of mentions of unpackaged?

Reply:
Correct. Other than the comments around packed vs unpackaged or UWP vs Win32.